src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/DateValidators.java

```diff
@@ -62,4 +64,21 @@ private static boolean isDateOnOrAfterAnotherDate(LocalDate aDate,
 
         return true;
     }
+
+    public static boolean isCeasedDateOnOrAfterIndividualsDateOfBirth(LocalDate trustCeasedDate,
+                                                                           List<TrustIndividualDto> individuals,
+                                                                           String qualifiedFieldName,
+                                                                           Errors errors,
+                                                                           String loggingContext) {
+
+        for(TrustIndividualDto individual : individuals) {
+            if (trustCeasedDate.isBefore(individual.getDateOfBirth())) {
+                setErrorMsgToLocation(errors, qualifiedFieldName, ValidationMessages.CEASED_DATE_BEFORE_INDIVIDUALS_DATE_OF_BRITH_ERROR_MESSAGE.replace("%s", qualifiedFieldName));
```

Comment:
Rather than use `.replace()` better to use `String.format()` here, which is designed to work with the `%s` placeholders. I would also do this once, assign to a string and reuse it in the `ApiLogger` output.

Reply:
Removed with the refactor above